packages/host/app/templates/card.hbs

```diff
@@ -1,4 +1,6 @@
-<this.getIsolatedComponent />
+<div class="card-isolated-component">
```

Comment:
Unrelated to this PR (and not requesting changes on this), but I really wish we had a utility css class approach. This way we could just use something like:

```
<div class="sp-lg">
  <this.getIsolatedComponent />
</div>
```

With this we could avoid inventing a new class almost every time we needed to add some padding to an element or similar basic stuff.

Reply:
Let's have another discussion about this during a boxel mob session or retro. Maybe we can just have some utility classes and restrict it for host app use. Utility classes idea may not work with css-encapsulation changes that are coming, though.